CareKit/CareContents/OCKCareContentsViewController.h

```diff
@@ -137,6 +140,24 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)careContentsViewController:(OCKCareContentsViewController *)viewController didActivatePullToRefreshControl:(UIRefreshControl *)refreshControl;
 
+/**
+ Asks the delegate for text to display below each `OCKCareCardButton` in the cell when in a selected state.
+ 
+ @param viewController              The view controller providing the callback.
+ @param interventionActivity        The `OCKCarePlanEvent` that the button represents
+ @param index                       The index of the `OCKCareCardButton` within the cell
+ */
+- (nullable NSString *)careContentsViewController:(OCKCareContentsViewController *)viewController selectedStateTextForInterventionActivity: (OCKCarePlanEvent *)interventionActivity atButtonIndex: (NSInteger)index;
```

Comment:
parameter name 'interventionActivity' of type OCKCarePlanEvent might be misleading.